config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/JmsDestinationEntityBuilder.java

```diff
@@ -0,0 +1,195 @@
+        } else {
+            contextPropertiesTemplateProps.put(JNDI_PASSWORD, jmsDestination.getJndiPassword());
+        }
+        
+        if (!isInbound) {
+            OutboundJmsDestinationDetail outboundDetail = jmsDestination.getOutboundDetail();
+            ReplyType replyType = outboundDetail.getReplyType();
+            jmsDestinationDetailProps.put(REPLY_TYPE, replyType);
+            
+            if (SPECIFIED_QUEUE.equals(replyType)) {
+                jmsDestinationDetailProps.put(REPLY_QUEUE_NAME, outboundDetail.getReplyToQueueName());
+            }
+
+            jmsDestinationDetailProps.put(USE_REQUEST_CORRELATION_ID, outboundDetail.useRequestCorrelationId());
+            jmsDestinationDetailProps.put(OUTBOUND_MESSAGE_TYPE, outboundDetail.getMessageFormat());
+
+            PoolingType poolingType = outboundDetail.getPoolingType();
+            if (CONNECTION.equals(poolingType)) {
+                contextPropertiesTemplateProps.put(CONNECTION_POOL_ENABLED, true);
+                ConnectionPoolingSettings connectionPoolingSettings = outboundDetail.getConnectionPoolingSettings();
+                if (connectionPoolingSettings != null) {
+                    if (connectionPoolingSettings.getSize() != null) {
+                        contextPropertiesTemplateProps.put(CONNECTION_POOL_SIZE, connectionPoolingSettings.getSize());
+                    }
+
+                    if (connectionPoolingSettings.getMinIdle() != null) {
+                        contextPropertiesTemplateProps.put(CONNECTION_POOL_MIN_IDLE, connectionPoolingSettings.getMinIdle());
+                    }
+
+                    if (connectionPoolingSettings.getMaxWaitMs() != null) {
+                        contextPropertiesTemplateProps.put(CONNECTION_POOL_MAX_WAIT, connectionPoolingSettings.getMaxWaitMs());
+                    }
+                }
+            } else {
+                contextPropertiesTemplateProps.put(CONNECTION_POOL_ENABLED, false);
+                SessionPoolingSettings sessionPoolingSettings = outboundDetail.getSessionPoolingSettings();
+                if (sessionPoolingSettings != null) {
+                    if (sessionPoolingSettings.getSize() != null) {
+                        contextPropertiesTemplateProps.put(SESSION_POOL_SIZE, sessionPoolingSettings.getSize());
+                    }
+
+                    if (sessionPoolingSettings.getMaxIdle() != null) {
+                        contextPropertiesTemplateProps.put(SESSION_POOL_MAX_IDLE, sessionPoolingSettings.getMaxIdle());
+                    }
+
+                    if (sessionPoolingSettings.getMaxWaitMs() != null) {
+                        contextPropertiesTemplateProps.put(SESSION_POOL_MAX_WAIT, sessionPoolingSettings.getMaxWaitMs());
+                    }
+                }
+            }
+        } else {
+            // (kpak) - inbound
+            // destinationDetailsProps.put("inbound.acknowledgementType", "");
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=138&issues=AWdc315Ny5Us1olWmrcf&open=AWdc315Ny5Us1olWmrcf)

[ISSUE_KEY]: AWdc315Ny5Us1olWmrcf